mssql-mixin/signals/database.libsonnet

```diff
@@ -0,0 +1,58 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+
+function(this)
+  {
+    filteringSelector: this.filteringSelector,
+    groupLabels: this.groupLabels,
+    instanceLabels: this.instanceLabels,
+    enableLokiLogs: this.enableLokiLogs,
+    aggLevel: 'none',
+    aggFunction: 'avg',
+    alertsInterval: '2m',
+    discoveryMetric: {
+      prometheus: 'mssql_io_stall_seconds_total',
+    },
+
+    signals: {
+      databaseWriteStallDuration: {
+        name: 'Database write stall duration',
+        nameShort: 'Write Stall',
+        type: 'raw',
+        description: 'Time spent waiting for database write operations.',
+        unit: 'seconds',
+        sources: {
+          prometheus: {
+            expr: 'increase(mssql_io_stall_seconds_total{%(queriesSelector)s, db=~"$db", operation="write"}[$__rate_interval:])',
```

Comment:
can `db` just added as the last element of instanceLabels? Can it be considered the smallest instnance you want to observ (single db within MSSQL host?) would it work?